data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+	`	Occasionally, Anna alternates between her left and right hand to sign books whenever the other is tired. For hours, this continues. You tried to keep count of how many copies she signed, but gave up when it reached six hundred.`
+	choice
+		`	"Are you left or right handed?"`
+	`	"I was originally born left-handed, but as a child I chose to learn how to use my right hand, because I thought being able to use both hands would be cool."`
+	`	The signing continues for what feels like four hours. Then, the same Kimek in the bookstore uniform announces that the store is to close within an hour.`
+	`	As the store closes and the queue shortens, Anna signs the last Kimek's copy. With that, the event is over. Anna then tells you to meet her at your ship. You and Anna walk into your ship, accompanied by Heliarchs and the occasional camera flash. You open the hatch to allow Anna inside, and you enter and lock it behind you before any obsessed fans or (Coalition equivalent of) paparazzi get the chance to break in.`
+	`	"Today was a resounding success. My book succeeded beyond my wildest dreams, and it was all because of you <first>!", she says, almost in a teary voice. She pulls out a credit chip. "Here, two million credits. You've done so much for me and my career, and you deserve it. Additionally, you will be receiving a salary of 5,000 credits daily, a small but significant portion of my royalties."`
+	branch "player understands coalition edition"
+		has "language: Coalition"
+	`	Anna then takes out another copy of "Beyond the Hyperdrive", which she signs in front of you. It is the Coalition edition of the book. The symbols are completely indecipherable to you. Just another book for your collection.`
+		accept
+	label "player understands coalition edition"
+		`	Anna then takes out another copy of "Beyond the Hyperdrive", which she signs in front of you. It is the Coalition edition of the book. The symbols are indecipherable to you, but your translation device helps you understand the contents. You realize that the book has been significantly altered from the version sold in human space.`
+		choice
+			`	"Anna, why is this book so different from the copies sold in human space?"`
+		`	Anna explains how strict the Coalition was in their censorship, having to continually make revisions to the book to make it "more suitable for the Coalition". The major plot points remained unchanged, but many parts of the book, particularly those about rebellion or the casual mention of ideas "unsuitable for the Coalition" were either deleted or heavily edited.`
+		choice
+			`	"Wait, that's how bad the censorship is?"`
+		`	"Yes. It caught me unexpectedly. On Zug, you can write whatever you want and you didn't have to worry about censorship. But here, the Heliarchs were breathing down my neck for the six months I had to stay to help revise the book."`
```

Comment:
By this point Anna has been in the Coalition for months, so she'd have picked up on the socially acceptable and socially unacceptable, to an extent. You should have her be more discrete in how she talks to you about the censorship, and what she discloses about the censorship. This applies to both how she acts during this part, and how she speaks during it too.